src/components/spinner/index.tsx

```diff
@@ -0,0 +1,15 @@
+import React, { Component } from 'react';
+
+import SC from './styled';
+
+const Spinner: React.FC = () => (
+  <SC.Spinner>
+    <SC.SpinnerImage />
+  </SC.Spinner>
+);
+
+export default class SpinnerComponent extends Component {
```

Comment:
Er det en grunn til at du gjør dette i stedet for å bare eksportere Spinner direkte?

Reply:
Fikser..